devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusBuild.java

```diff
@@ -140,6 +159,26 @@ public void buildQuarkus() {
         }
     }
 
+    private void exportCustomManifestProperties(Properties buildSystemProperties) {
+        if (this.manifest == null) {
+            return;
+        }
+
+        for (Map.Entry<String, Object> attribute : manifest.getAttributes().entrySet()) {
+            buildSystemProperties.put(
+                    String.format("%s.\"%s\"", MANIFEST_ATTRIBUTES_PROPERTY_PREFIX, attribute.getKey()),
```

Comment:
Probably safer with some escaping for `"` in the keys (I know it shouldn't happen but let's still handle it if we can).

Reply:
BTW, if `"` is not supported in keys, one way to handle it is to throw an error if there is one.